source/pdk/1.0/pdk_troubleshooting.md

```diff
@@ -5,9 +5,18 @@ canonical: "/pdk/1.0/pdk_troubleshooting.html"
 description: "Troubleshooting Puppet Development Kit, the shortest path to developing better Puppet code."
 ---
 
+{:.concept}
+## PDK not in shell PATH
+ 
+PDK is not automatically added to the PATH in some shells. To fix this, add the PATH to the affected shells:
```

Comment:
With this being zsh/OSX specific this could use some rewording

Reply:
Will do! I wasn't sure if it still applied to shells we might not know about.